app/llm_wrapper.py

```
@@ -10,131 +10,68 @@
+async def ask_local(messages: List[Dict[str, str]]) -> str:
+    async with aiohttp.ClientSession() as session:
+        async with session.post(
+            "http://llama:8000/v1/chat/completions",
+            headers={
+                "Content-Type": "application/json"
+            },
+            json={
+                "model": llm_model,
+                "messages": messages,
+            }
+        ) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)